### PR TITLE
Bugfix: hitAreaCallback is not a function

### PR DESCRIPTION
**Pull request by ArifZx**
Please do not update the README or Change Log, we will do this when we merge your PR.

This PR (delete as applicable)

* Fixes a bug

Describe the changes below:

Screen Shot 2022-01-31 at 09 38 45

Screen Shot 2022-01-31 at 09 40 02

### CODE REVIEW

File: src/input/InputManager.js [hunk 1]
```diff
@@ -971,8 +971,9 @@ var InputManager = new Class({
         y += gameObject.displayOriginY;
 
         var input = gameObject.input;
+        var hitAreaCallback = input?.hitAreaCallback;
```
Comment (natureofcode) [inline src/input/InputManager.js]: Unfortunately, **Optional chaining operator (?.)** is not **ES5**-compliant, so it can't be used in Phaser 3 code.

**Comment by samme** [comment]
What's the bug?

File: src/input/InputManager.js (hunk 1 above)
Comment (ArifZx) [reply src/input/InputManager.js]: got it noted and updated

**Comment by ArifZx** [comment]
> What's the bug?

sometimes hitAreaCallback is not a function

**Comment by samme** [comment]
Sometimes how? How do you produce the bug?

**Comment by ArifZx** [comment]
> Sometimes how? How do you produce the bug?

Still don't know to reproduce that, but found causes hitAreaCallback is undefined
Screen Shot 2022-02-02 at 16 30 40

Maybe after clear interactive, but the object still in queue (_list)

**Comment by samme** [comment]
I just don't see how it could happen. In [clear()](https://newdocs.phaser.io/docs/3.55.2/Phaser.Input.InputPlugin#clear) `gameObject.input` is set to null.

**Comment by photonstorm** [comment]
Sorry but I'm going to close this until an example can be provided showing how it fails. The error is quite clear IMHO, in that a function wasn't provided, or that you cleared the input and then still tried to call an input event. Please provide an example, thanks.

**Pull request closed without merging**